Diary/Controller/DiaryDetailViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  DiaryDetailViewController.swift
+//  Created by Minseong, Lingo
+//
+
+import UIKit
+
+final class DiaryDetailViewController: UIViewController {
+  private let titleLabel = UILabel()
+  private let bodyTextView = UITextView()
+  private let diary: Diary
+
+  init(diary: Diary) {
+    self.diary = diary
+    super.init(nibName: nil, bundle: nil)
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.configure()
+    self.setNotification()
+  }
+
+  override func viewDidDisappear(_ animated: Bool) {
+    super.viewDidDisappear(animated)
+    self.removeNotification()
+  }
+
+  private func configure() {
+    self.title = diary.createdAtString
+    self.view.backgroundColor = .systemBackground
+
+    self.titleLabel.font = .preferredFont(forTextStyle: .headline)
+    self.titleLabel.adjustsFontForContentSizeCategory = true
+    self.titleLabel.text = diary.title
+
+    self.bodyTextView.font = .preferredFont(forTextStyle: .body)
+    self.bodyTextView.adjustsFontForContentSizeCategory = true
+    self.bodyTextView.text = diary.body
+
+    let container = UIStackView(arrangedSubviews: [titleLabel, bodyTextView])
+    container.axis = .vertical
+    container.spacing = 20.0
+
+    self.view.addSubview(container)
+    container.translatesAutoresizingMaskIntoConstraints = false
+    NSLayoutConstraint.activate([
+      container.topAnchor.constraint(
+        equalTo: view.safeAreaLayoutGuide.topAnchor,
+        constant: 8.0
+      ),
+      container.bottomAnchor.constraint(
+        equalTo: view.safeAreaLayoutGuide.bottomAnchor
+      ),
+      container.leadingAnchor.constraint(
+        equalTo: view.safeAreaLayoutGuide.leadingAnchor,
+        constant: 8.0
+      ),
+      container.trailingAnchor.constraint(
+        equalTo: view.safeAreaLayoutGuide.trailingAnchor,
+        constant: -8.0
+      )
+    ])
+  }
+
+  private func setNotification() {
+    NotificationCenter.default.addObserver(
+      self,
+      selector: #selector(keyboardWillShow(_:)),
+      name: UIResponder.keyboardWillShowNotification,
+      object: nil
+    )
+
+    NotificationCenter.default.addObserver(
+      self,
+      selector: #selector(keyboardWillHide(_:)),
+      name: UIResponder.keyboardWillHideNotification,
+      object: nil
+    )
+  }
+
+  private func removeNotification() {
```

Comment:
[P2] 어떤 역할을 하는 메서드인지 불분명합니다. 내부 구현을 참고해 보면 `removeKeyboardNotificationObesrvation()` 정도가 적절해 보입니다.

Reply:
[공식문서](https://developer.apple.com/documentation/foundation/notificationcenter/1415360-addobserver)의 설명에서 아래의 내용을 확인했습니다.

If your app targets iOS 9.0 and later or macOS 10.11 and later, you do not need to unregister an observer that you created with this function. If you forget or are unable to remove an observer, the system cleans up the next time it would have posted to it.

따라서, 해당 메서드를 제거했습니다.